airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -42,7 +42,7 @@
 - name: Apify Dataset
   sourceDefinitionId: 47f17145-fe20-4ef5-a548-e29b048adf84
   dockerRepository: airbyte/source-apify-dataset
-  dockerImageTag: 0.1.3
+  dockerImageTag: 0.1.7
```

Comment:
should these be reverted before we merge?

Reply:
I've been publishing the version bumps while testing, do you think it's worth reverting and removing the published bumps?

Reply:
Ah should we just add that info in the changelog just so we know what these versions are?

Reply:
Yes good point, will add that